pyproject.toml

```diff
@@ -50,5 +47,13 @@ Issues = "https://github.com/DemocracyClub/dc_django_utils/issues"
 requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
 
-[tool.black]
+[tool.ruff]
 line-length = 80
+lint.ignore = ["E501"]
+lint.extend-select = [
+    "I",
+    "C4",
+    "SIM",
+    "Q003",
+    "RET",
+]
```

Comment:
This is the standard config we should be using
See https://docs.google.com/document/d/1Sb_m86KG5ca0lOZMgtJZwch32eVfA0syAY9MeIW1Ljk/edit#heading=h.gnuw7hn63sl7
Note that adding the config here is the reason we now have a load of new errors that need fixing (mostly import sorting)